From f6ae3448e04a71c5463fc0a77a4ad9c8482ef9f5 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Fri, 25 Dec 2020 14:18:23 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: set ACLK_VOP to 500M for
 rk3568"

ACLK_VOP is assigned by cru now.

This reverts commit de0105f86b373d93e7aafc136b4b33fdb2f0cb88.

Change-Id: I9b9390c444d215eaf940053d8617f28b3632e6a9
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 9cdb46417123..cc54c57db71a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -1391,8 +1391,6 @@
 		interrupts = <GIC_SPI 148 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru ACLK_VOP>, <&cru HCLK_VOP>, <&cru DCLK_VOP0>, <&cru DCLK_VOP1>, <&cru DCLK_VOP2>;
 		clock-names = "aclk_vop", "hclk_vop", "dclk_vp0", "dclk_vp1", "dclk_vp2";
-		assigned-clocks = <&cru ACLK_VOP>;
-		assigned-clock-rates = <500000000>;
 		iommus = <&vop_mmu>;
 		power-domains = <&power RK3568_PD_VO>;
 		status = "disabled";
-- 
2.35.3

